util/s3/s3Helper.go

```diff
@@ -247,7 +247,7 @@ func GetVersionFromName(name string) (version uint64, err error) {
 	var val uint64
 	for index, match := range submatchAll[0] {
 		if index > 0 {
-			version <<= 16
+			version <<= 24
```

Comment:
Is there a backwards compatibility issue? Hopefully these patch numbers transient?

Reply:
thank you for pointing out, it is used in `GetVersionPartsFromVersion` to back format human version, fixed + tests